packages/core/src/domain/session/sessionCookieStore.spec.ts

```diff
@@ -21,13 +21,15 @@ describe('session cookie store', () => {
     cookieStorage.persistSession(sessionState)
     const session = cookieStorage.retrieveSession()
     expect(session).toEqual({ ...sessionState })
+    expect(document.cookie).toMatch(/_dd_s=.*id=.*created/)
   })
 
   it('should delete the cookie holding the session', () => {
     cookieStorage.persistSession(sessionState)
     cookieStorage.clearSession()
     const session = cookieStorage.retrieveSession()
     expect(session).toEqual({})
+    expect(document.cookie).not.toMatch(/_dd_s=/)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about using `getCookie(SESSION_COOKIE_NAME)` for those checks?